urbanopt_to_alfalfa.ipynb

```diff
@@ -0,0 +1,300 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "dfae0c8b",
+   "metadata": {},
+   "source": [
+    "# URBANopt to Alfalfa workflow "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c2de5cb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import shutil\n",
+    "import json\n",
+    "import pandas\n",
+    "import datetime\n",
+    "import time\n",
+    "from alfalfa_client.alfalfa_client import AlfalfaClient\n",
+    "from pprint import pprint\n",
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f892da4d",
+   "metadata": {},
+   "source": [
+    "### Define alfalfa client object\n",
+    "\n",
+    "Use `pip install alfalfa-client` prior to running further commands"
```

Comment:
I'm not sure if we need this since the installation instructions are in the README.md

Reply:
Can you point me to where these instructions are in the documentation? I will link to those.